client/dashboard/page-layout/index.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+	__experimentalHeading as Heading,
+	__experimentalText as Text,
+} from '@wordpress/components';
+import './style.scss';
+
+const sizes = {
+	large: {
+		maxWidth: '1200px',
+	},
+	small: {
+		maxWidth: '600px',
+	},
+};
+
+function PageLayout( {
```

Comment:
Is this duplicate `PageLayout` component used anywhere? Or maybe is it simply a bad merge.

Reply:
Good catch, thanks!